### PR TITLE
Telnet python 3

### PR DESCRIPTION
**Pull request by xandfury**
[Build Status](https://travis-ci.org/xandfury/telnetsrvlib) [Coverage Status](https://coveralls.io/github/xandfury/telnetsrvlib?branch=master) [Codacy Badge](https://www.codacy.com/app/xandfury/telnetsrvlib?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=xandfury/telnetsrvlib&amp;utm_campaign=Badge_Grade)

Initial work on telnet lib.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+language: python
+python:
+- '3.5'
```
Comment (glaslos) [inline .travis.yml:3]: what about 3.6?

File: telnetsrv/green.py [hunk 2]
```diff
@@ -1,20 +1,23 @@
-#!/usr/bin/python
+"""
 # Telnet handler concrete class using green threads
+"""
 
-import gevent, gevent.queue
+import gevent
```
Comment (glaslos) [inline telnetsrv/green.py:5]: we probably want to replace gevent with asyncio

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py35
```
Comment (glaslos) [inline tox.ini]: 3.6?

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+skipsdist = True
+usedevelop = True
+
+[testenv]
+basepython =
+    py35: python3.5
```
Comment (glaslos) [inline tox.ini]: 3.6?

**Review by glaslos — CHANGES_REQUESTED** [review]

File: telnetsrv/green.py (hunk 2 above)
Comment (xandfury) [reply telnetsrv/green.py:5]: Hey Lukas. Actually Daniel and I thought of using fork for Conpot's telnet server. So the choice of py35 and gevent are derived from there. I have added py36 support though.
It only makes sense to have multiple handlers if we plan to use this fork in multiple projects. 

cc: @creolis What do you think?

**Comment by xandfury** [comment]
@glaslos @creolis On a side note, please refrain from squashing commits. I think one of these days someone might do a git bisect to look for potential errors.

File: telnetsrv/green.py (hunk 2 above)
Comment (glaslos) [reply telnetsrv/green.py:5]: @xandfury Sorry, my bad, stick to gevent for now

**Comment by glaslos** [comment]
LGTM

**Review by glaslos — APPROVED** [review]

**Pull request merged**